apps/web/src/server/applications/case/documentation-metadata/documentation-metadata.controller.js

```diff
@@ -252,5 +252,25 @@ const getLayoutParameters = (requestParameters, responseLocals) => {
 		return null;
 	}
 
+	if (layout.metaDataName === 'documentType' && layout.items) {
+		layout.items = layout.items.map((item) => {
+			if (item.value === 'Rule 6 letter') {
+				return {
+					...item,
+					text: caseData.isMaterialChange ? 'Regulation 27 and 28 letter' : 'Rule 6 letter'
+				};
+			}
+
+			if (item.value === 'Rule 8 letter') {
```

Comment:
I think an `else if` would be better here to signal that we either want the first if statement or this to be true (as both cant be true) but feel free to ignore

Reply:
Thanks Kartikay, I've made the change.